roles/variables/defaults/main.yml

```diff
@@ -1240,9 +1240,6 @@ control_center_custom_properties: {}
 ### Boolean used for disabling of systemd service restarts when rootless install is executed
 control_center_skip_restarts: "{{ skip_restarts }}"
 
-### Boolean used for enabling OAuth on Control Center
-control_center_oauth_enabled: "{{ oauth_enabled }}"
```

Comment:
what will be default value of control_center_oauth_enabled?

Reply:
there is no control_center_oauth_enabled because server level oauth in c3 is sso, which we enable with sso_enabled